### PR TITLE
Creating Bubbles via graph

### PR DESCRIPTION
**Pull request by Dletta**
- [ ] switch back to normal view
- [ ] hook up correct events
- [ ] move them a bit closer 
- [ ] test many more people

### CODE REVIEW

File: src/assets/css/app.css [hunk 1]
```diff
@@ -717,6 +717,37 @@ input:checked + .slider:before {
   background-color: #673ab7;
 }
 
+/* graphed Video Bubbles
```
Comment (QVDev) [inline src/assets/css/app.css:720]: Don't know if its overhead or decrease loading time. Maybe we can put this in a different CSS file? To keep things seperate?

File: src/assets/js/graphThing.js [hunk 2]
```diff
@@ -1,19 +1,160 @@
...
+      document.getElementById('grid').appendChild(div);
+    }
+    med.graphedVideo = true;
+    self.graphWorker = new Worker('/assets/workers/workerGraph.js');
+    /* Prototype
+    *
+    *
+    *
+
+    // create Graph Div to make the graph area??
```
Comment (QVDev) [inline src/assets/js/graphThing.js:64]: Think we can remove dead code, unless you really going to use this in the future we can keep and uncomment

File: src/assets/js/sfu/video.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ export default class Video {
     webrtc.on("readyToCall", function (peerCount) {
       // webrtc.joinRoom(self.getRoom());
       webrtc.startBroadcast(peerCount);
+      med.ee.emit("sfu:peer");
```
Comment (QVDev) [inline src/assets/js/sfu/video.js:49]: Do we need this also in the mesh.js?

**Comment by Dletta** [comment]
Commiting into bubble branch

**Pull request merged**